OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -136,6 +139,13 @@ void LiveFunctionsDataView::DoSort() {
 //-----------------------------------------------------------------------------
 const std::string LiveFunctionsDataView::MENU_ACTION_SELECT = "Hook";
 const std::string LiveFunctionsDataView::MENU_ACTION_UNSELECT = "Unhook";
+const std::string LiveFunctionsDataView::MENU_ACTION_GO_TO_FIRST =
+    "Go to First";
+const std::string LiveFunctionsDataView::MENU_ACTION_GO_TO_LAST = "Go to Last";
+const std::string LiveFunctionsDataView::MENU_ACTION_GO_TO_SHORTEST =
```

Comment:
I think the naming here should be in sync with the naming in the table (min/max).

Reply:
Yes, I though about that but for some reason I wrote shortest/longest. Will change it!